packages/explorer/src/app/(explorer)/error.tsx

```diff
@@ -10,13 +10,19 @@ type Props = {
   reset: () => void;
 };
 
-export default function Error({ reset }: Props) {
+export default function Error({ reset, error }: Props) {
   const getUrl = useWorldUrl();
   return (
     <main className="px-6 py-24 text-center">
       <p className="text-3xl font-semibold text-orange-600">400</p>
       <h1 className="mt-4 text-3xl font-bold tracking-tight text-white sm:text-5xl">Something went wrong :(</h1>
 
+      {error.message && (
+        <pre className="mt-8 inline-block w-[580px] whitespace-normal rounded-md border border-red-500 bg-red-50 p-4 text-left font-mono text-sm leading-6 text-red-500">
```

Comment:
feels a little too contrasty with the black theme

Reply:
was not big fan of the result either, played around a bit more, looks better now imo 
![CleanShot 2024-09-03 at 21 24 01@2x](https://github.com/user-attachments/assets/3b7d2b97-ee84-4cf5-8990-3bcb1e482ed0)

Reply:
works for me! thanks